src/Features/ERDDAP/Platform/Observations/Condition/index.tsx

```diff
@@ -46,8 +57,28 @@ export const ErddapObservedCondition: React.FunctionComponent<Props> = ({ platfo
           <h4>
             {ts.data_type.long_name} {depth} <Info timeSeries={[ts]} id={index} startDate={startDate} />
           </h4>
-          <UseDataset timeSeries={ts} startTime={startDate}>
-            {({ dataset }) => <ChartTimeSeriesDisplay {...{ dataset, standardName, unitSystem }} timeSeries={ts} />}
+          {index === 0 && (
+            <div>
+              <Button color="light" outline onClick={() => toggle()} className="timeframe-collapse-button">
+                <p style={{ marginBottom: 0, marginRight: "5px", color: "#083d52" }}>Change Timeframe</p>
+                <Calendar width={"20px"} height={"20px"} />
+              </Button>
+              <Collapse isOpen={isOpen} className="timeframe-collapse">
+                <div className="observation-timeframe-selection">
+                  <TimeframeSelector graphFuture={false} />
```

Comment:
Maybe the `TimeframeSelector` is an is part of the `<h4>` (or really a parent component of it), rather than trying to be manipulated with CSS to play nice at all sizes?

Reply:
It happens when navigating from the current conditions page via the card headers for any of those conditions, and also if you click the 'last values' between the station info and the unit system. 

https://github.com/user-attachments/assets/6f3ff55f-dfd1-4ba5-aff9-fa81672f2960

I think the issue is that it's getting nulls or empty strings when start and end params aren't set, so we may want to add fallback values https://github.com/gulfofmaine/Neracoos-1-Buoy-App/pull/3194/files#diff-b0b4393f4c4f58be180f325dd4511e323a5fa8ca0522da583515f424a0629467R40-R41

Let's not add params to links by default and have them bare. https://github.com/gulfofmaine/Neracoos-1-Buoy-App/pull/3194/files#diff-0301babb06bd3617e76d92b6de2baf63f786c2a63259bd8ce98600660a4b9e6bR78-R116

Looks and behaves better, but maybe a touch of padding/margin (I can't ever remember which is which side).

<img width="725" alt="image" src="https://github.com/user-attachments/assets/9233485f-1b56-4a9c-9ada-be7c65b405c0">

Reply:
The most recent commits should take care of
- The margin between the title and the timeframe selector (and aligns to the center so it doesn't look too terrible)
- Default time params so we don't change the link path, but set default values for start and end, and only when the user changes the timeframe should the params in the url change. 

Thanks for the feedback on all of this! Keep it comin!